pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -279,6 +279,12 @@ private void internalRetryableDeleteNamespaceAsync0(boolean force, int retryTime
                                         return old;
                                     });
                                 }
+                                allUserCreatedTopics.removeIf(t ->
```

Comment:
maybe we can use Set here. removeIf in List seems may cause more CPU.

Reply:
addressed